python-bindings/pythonbindings.cpp

```diff
@@ -362,7 +377,10 @@ PYBIND11_MODULE(blspy, m)
         .def(
             "__bytes__",
             [](const G1Element &ele) {
-                vector<uint8_t> out = ele.Serialize();
+                vector<uint8_t> out;
+                Py_BEGIN_ALLOW_THREADS
```

Comment:
the only way this makes sense to me is if `Py_BEGIN_ALLOW_THREADS` introduces a new scope. Does it?

Reply:
It does